UniConnectApp/app/src/test/java/com/example/uniconnect/UniConnectUnitTest.kt

```diff
@@ -91,6 +92,13 @@ class UniConnectUnitTest {
     private fun thenTheUniversityCollectionSizeShouldBeGreaterThanZero() {
         assertNotNull(allUniversities)
         assertTrue(allUniversities!!.isNotEmpty())
+        var containsMarywoodUniversity = false
+        allUniversities!!.forEach {
+            if (it.name.equals("Marywood University") && it.code.equals("US")) {
+                containsMarywoodUniversity = true
+            }
+        }
+        assertTrue(containsMarywoodUniversity)
```

Comment:
This checks that actual results were pulled because it is possible to get an array of empty string that will still count as not empty